mobile/src/feeds/home-feeds-screen.tsx

```diff
@@ -47,60 +50,82 @@ const mapDispatchToProps = (dispatch: Dispatch<any>): FeedScreenDispatchProps =>
     fetchOldFeeds: () => dispatch(fetchOldFeeds()),
 });
 
+//============================================================================
 class HomeFeedsScreenImpl extends React.Component<FeedsScreenProps & FeedScreenDispatchProps> {
     public static navigationOptions = navBar.configurate();
 
+    //----------------------------------------------------------------------------
     public render() {
         const feeds = this.sortedFeeds();
 
         return this.props.feeds.size > 0 ?
-                <FlatList
-                    style={styles.view}
-                    keyExtractor={this.keyExtractor}
-                    ItemSeparatorComponent={this.itemSeparator}
-                    data={feeds}
-                    extraData={this.props.employees}
-                    renderItem={this.renderItem}
-                    onEndReached={this.endReached}
-                    onEndReachedThreshold={0}
-                    refreshing={false}
-                    onRefresh={this.onRefresh}
-                />
-        : <LoadingView/>;
+            <FlatList
+                style={styles.view}
+                keyExtractor={this.keyExtractor}
+                ItemSeparatorComponent={this.itemSeparator}
+                data={feeds}
+                extraData={this.props.employees}
+                renderItem={this.renderItem}
+                onEndReached={this.endReached}
+                onEndReachedThreshold={0.2}
+                refreshing={false}
+                onRefresh={this.onRefresh}
+                ListFooterComponent={this.footer}
+            /> :
+            <LoadingView/>;
     }
 
+    //----------------------------------------------------------------------------
+    // noinspection JSMethodCanBeStatic
```

Comment:
Why not to actually make it static?

Reply:
I would rather make them lambdas

Reply:
done: 9ade2597c215d48be39585e79cec5fdc7193608e